test/unit/AdditionalDetailsViewer.test.js

```diff
@@ -1,10 +1,12 @@
 import React from 'react';
 import { expect } from 'chai';
-import { shallow, mount } from 'enzyme';
+import Enzyme, { shallow, mount } from 'enzyme';
+import Adapter from 'enzyme-adapter-react-16';
+
+Enzyme.configure({ adapter: new Adapter() });
```

Comment:
You don't need this more than once. I suggest putting it in the general helper test file `/src/testHelper`. I don't think it matters where in the file it's located but most conventions for the adapter are at the top of the file after the imports.